benchmarks/transport_montecarlo_numba_formal_integral_p.py

```diff
@@ -84,16 +111,11 @@ def time_explosion(self):
     @parameterize({"p": [0.0, 0.5, 1.0], "Test data": TESTDATA})
     def time_calculate_z(self, p, test_data):
         func = formal_integral.calculate_z
-        inv_t = 1.0 / self.time_explosion
-        len(self.formal_integral_geometry(test_data).r_outer)
+        inv_t = 1.0 / self.formal_integral_model.time_explosion
```

Comment:
Can you recheck if this is the right path for time_explosion?

Reply:
done